web/src/components/core/MenuHeader.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) [2023-2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Flex, Content } from "@patternfly/react-core";
+import spacingStyles from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+
+/**
+ * Renders the content to be used at a menu entry describing subsequent options.
+ * @component
+ *
+ * @param title - Short sentence describing the functionality.
+ * @param descriptions - Extra details rendered with a smaller font.
+ */
+export default function MenuHeader({ title, description }): React.ReactNode {
```

Comment:
Please, define a `MenuHeaderProps` type.

Reply:
Or simply add the types

```suggestion
export default function MenuHeader({ title, description }: { title: string, description: string) {

```

Reply:
NP: BTW, as said in another review, until now we have been omitting types that TypeScript is able to infer.